daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -0,0 +1,94 @@
+- functionally correct and robust
+- well documented and maintained
+
+To make sure a component conforms to the standards set by Dapr, there are a set of tests run against a component in a Dapr maintainers managed environment. Once the tests pass consistently for a certain number of runs, the maturity level can be determined for a component. 
+
+## Certification levels 
+
+The proposed levels are as follows:
+- [Alpha](#alpha)
+- [Beta](#beta)
+- [General availability (GA)](#general-availability-ga)
+
+### Alpha
+
+- Component implements the required interface and works as described in the specification
+- Component has documentation
+- Component might be buggy or might expose bugs on integration
+- Component may not pass all conformance tests
+- Component may not have conformance tests
+- Not recommended for use in production
+
+All components generally start at the Alpha stage.
+
+### Beta
+
+- Component must pass all the component conformance tests defined to satisfy the component specification
+- The component conformance tests have been run in a Dapr maintainers managed environment
+- Component contains a record of the conformance test result reviewed and approved by Dapr maintainers with specific components-contrib version
+- Not recommended for use in production
+
+### General Availability (GA)
+
+- Has at least two different users in production
+- A GA component has a defined maintainer in the Dapr community or the Dapr maintainers
+- Component is well documented, tested and maintained across multiple versions of components-contrib repo
+
+## Conformance tests 
+
+Each component in the [components-contrib](https://github.com/dapr/components-contrib) repository needs to adhere to a set of interfaces defined by Dapr. Conformance tests are tests that are run on these component definitions with their associated backing services such that the component is tested to be conformant with the Dapr interface specifications. 
+
+The conformance tests are defined for the following building blocks: 
+
+- State store
+- Secret store
+- Bindings 
+- Pub/Sub
+
+To know more about them please see the readme [here](https://github.com/dapr/components-contrib/blob/master/tests/conformance/README.md).
+
+### Test requirements
+
+- The tests should validate the functional behavior and robustness of component based on the component specification
+- All the details needed to reproduce the tests will be added as part of the component conformance test documentation
+
+## Component certification process
+
+### New component certification
+
+For a new component a request for component certification follows these steps:
+- This applies for either certifying a component as Beta from Alpha or GA from Beta stages
+- An issue is created with a request for certification of the component with the current and the new certification levels
+- The component owner/maintainer submits a PR for integrating the component to run with the defined conformance test suite
```

Comment:
How do we know who "owns" a component? Is there a owners file somewhere?

Reply:
This was my question. Is is worth having tracking issues (which are closed by can be updated) for each component? Or is this best in docs? The advantage of the issue, is that this maintains the history.

Reply:
I think having an issue tracking who own what component will be beneficial ... For tracking and maintaining history.
Or one more idea, we could have in each component after the license a comment line containing the maintainers of a component. It lives with the code and would evolve with the code then.

Reply:
What about an owners.txt file in the component's directory?

Reply:
Or `maintainers.txt` since `owners` might not be good terminology in open source.